spec/app/models/mdm/web_page_spec.rb

```diff
@@ -3,6 +3,48 @@
 
   context 'associations' do
     it { is_expected.to belong_to(:web_site).class_name('Mdm::WebSite') }
+
+    context 'serialized cookie attribute' do
+      let(:valid_cookie) do
+        {
+          name: 'test name',
+          value: 'test value'
+        }
+      end
+
+      let(:invalid_cookie) do
```

Comment:
fair point -- going put this back in development and see where we use those web page cookies